### ISSUE TITLE
App consuming battery in background?

### ISSUE THREAD

**Issue by opusforlife2** [NeoApplications/Neo-Store#617]
LOS 21 (A14) lets you see more and more specific time periods of battery usage. I was surprised to note my phone turning on battery saver, and checked battery usage. Neo Store seems to have been active for over an hour in the background, showing that it consumed 30% share of battery used by all apps in that time.

I can't think of a reason for this to occur. Auto updates are off. Repo updates are set to happen only once a day, and I already saw that happen earlier in the day.

**Comment by machiav3lli** [comment]
Was there any notificatoon stuck? (Cause notifications consume irrational amount of battery in my experience)

**Comment by machiav3lli** [comment]
Hm, I don't really have an idea what could be causing this. Did it happen again since then? Another facts that would be interesting: What were you interactions with the phone in that hour? Did you uninstall any app between that hour and checking things? Otherwise I can only take a guess that the battery manager is hallucinating (there's notes all over the place that in some instances the statistics are not that precise).

For reference I get consistently under 5% consumption for NS, with 4 hours sync interval (almost 4 auto-syncs a day considering sleep and work), opening the app myself once or twice a day.

**Comment by machiav3lli** [comment]
Hm… I can't really see what could be causing the long background times (for reference, by me it's regularly around 10 minutes a day), my only guess is the root installer (referencing the mystery in other issue #607)… when we can confirm this, then I'll need to check if there's any changes in the Magisk APIs resulting this…

**Comment by machiav3lli** [comment]
Yup, this would be a first step maybe. Are there any other non-default settings, that you have changed and would suspect?
